test/test_doc.py

```diff
@@ -134,6 +134,34 @@ class Foo(metaclass=FooMeta):
         assert x.docstring == ""
 
 
+@pytest.mark.parametrize(
+    "member, defaults",
+    [
+        (
+            "Foo",
+            {
+                "a": "1",
+                "b": "2",
+            },
+        ),
+        (
+            "PydanticStyleDataclass",
+            {
+                "a": "1",
+            },
+        ),
+    ],
+)
+def test_with_pydantic(member, defaults):
+    m = Module.from_name("test.with_pydantic")
+    assert "Foo" in m.members
+
+    cls: Class = m.members[member]
+
+    for k, v in defaults.items():
+        assert cls.members[k].default_value_str == v
```

Comment:
Please add a snapshot test instead:

1. Add Python file to `test/testdata`, e.g. `test/testdata/pydantic.py`
2. Register it [here](https://github.com/mitmproxy/pdoc/blob/ea2b69a4162226b07f68cd25d92d4d92dbbafaf8/test/test_snapshot.py#L97).
3. `uv run test/test_snapshot.py`

Reply:
Sounds good.